dnf-behave-tests/features/repoquery/main.feature

```diff
@@ -349,6 +349,29 @@ Given I successfully execute rpm with args "-i --nodeps {context.dnf.fixturesdir
       dummy-1:1.0-1.x86_64
       """
 
+   # --extras: installed pkgs, different NEVRA in available repository
+Scenario: dnf repoquery --extras (when there are such pkgs with different NEVRA in repository)
+ When I execute rpm with args "-i --nodeps {context.dnf.fixturesdir}/repos/dnf-ci-fedora/x86_64/wget-1.19.5-5.fc29.x86_64.rpm"
```

Comment:
Feature files in `features/repoquery` have a dedicated repository for each of them. Could you implement these scenarios using packages from that repository (in this case `repoquery-main`), so that it is not dependent on the globaly used `dnf-ci-fedora`?

It seems these tests require two repositories, I think adding `repoquery-main-2` is ok (the only thing you need for that now is spec files under `fixtures/specs/repoquery-main-2`).

One of the advantages this has is you're not influencing other tests when changing data for this feature file, like you have to do in this PR when you used `wget` from `dnf-ci-fedora`.

Reply:
I would keep it like it is. It increases litle bit complexity of other scenario, therefore it is also beneficial. Can we merge it like it is?